README.md

```diff
@@ -34,30 +34,40 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML allows us to code information and content into a webpage beyond presentation. We create elements with this language.
+
```

Comment:
Semantic HTML is simply html tags that have meaning built into them so you know what type of content it is.  Examples are <header> <footer> <article> etc.  <div> is not semantic, because you don't know what type of content it includes.